locale/en_US/LC_MESSAGES/django.po

```diff
@@ -2093,10 +1991,10 @@ msgstr "Change Email"
 #: mafiasi/registration/templates/registration/account.html:92
 msgid ""
 "Note: You can \"change\" your password to the current password to "
-"synchronize it with other services (e.g. Jabber)"
```

Comment:
This also requires a change in the registration/account.html template. (Actually, the note can be removed there, we don't need any kind of synchronization anymore.)